README

```diff
@@ -27,4 +27,5 @@ AUTHOR:
         Lennart Poettering with major support from Kay Sievers
 
 LICENSE:
-        GPLv2+ for all code.
+        GPLv2+ for all code,
+        CC-BY-3.0-US for the desktop icon (from the GNOME Project).
```

Comment:
Do we need a copy of the license text in the repo for this?

Reply:
Good question. To be honest, I am not sure, but almost every project has something licensed under CC and none of them has a full license copy in their repo. Take a look for example at [GNOME Software](https://gitlab.gnome.org/GNOME/gnome-software) or [Showtime](https://gitlab.gnome.org/GNOME/showtime). Both of them have CC licensed parts (AppStream metadata, icons etc.), but none of them has a full CC license copy in their archive. Including the [SPDX](https://spdx.org/licenses/) license identifier in README should be sufficient in my opinion.